src/ui/components/Widgets/Chart/hooks/useLoadingChart.ts

```diff
@@ -534,6 +534,20 @@ export const useLoadingChart = (props: LoadingChartHookProps) => {
         enable: !isInit && !canBeLoaded,
     });
 
+    /**
+     * cancel active requests when component is unmounted (changing dash tab, switching to another page)
+     */
+    React.useEffect(() => {
```

Comment:
Why we duplicating this logic ?
https://github.com/datalens-tech/datalens-ui/blob/f7953bb11cbc6227105f3c00bcd832ea43add40d/src/ui/components/Widgets/Chart/hooks/useLoadingChart.ts#L319-L320